src/GuideAtom.tsx

```diff
@@ -1,33 +1,255 @@
 import React from 'react';
 
 import { GuideAtomType } from './types';
+import { css, cx } from 'emotion';
+import { body, textSans } from '@guardian/src-foundations/typography';
+import {
+    news,
+    neutral,
+    opinion,
+    culture,
+    lifestyle,
+    labs,
+} from '@guardian/src-foundations/palette';
+import { SvgInfo } from '@guardian/src-icons';
+import { Footer } from './components/Footer';
+import { Summary } from './components/Summary';
 
-const Body = ({ html }: { html: string }) => (
-    <div
-        dangerouslySetInnerHTML={{
-            __html: html,
-        }}
-    />
+let linkColourStyle = news[300];
+let linkColourStyleHover = news[400];
+
+const containerStyling = css`
+    display: block;
+    position: relative;
+    margin-bottom: 0.75rem;
+    margin-top: 1rem;
+`;
+
+const detailStyling = css`
+    margin: 16px 0 36px;
+    background: ${neutral[93]};
+    padding: 0 5px 6px;
+    border-image: repeating-linear-gradient(
+            to bottom,
+            ${neutral[86]},
+            ${neutral[86]} 1px,
+            transparent 1px,
+            transparent 4px
+        )
+        13;
+    border-top: 13px solid black;
+    position: relative;
+    summary {
+        list-style: none;
+        margin: 0 0 16px;
+    }
+
+    summary::-webkit-details-marker {
+        display: none;
+    }
+
+    summary:focus {
+        outline: none;
+    }
+`;
+
+const bodyStyling = css`
+    ${body.medium()}
+    p {
+        margin-bottom: 0.5rem;
+    }
+
+    ol {
+        list-style: decimal;
+        list-style-position: inside;
+        margin-bottom: 1rem;
+    }
+
+    ul {
+        list-style: none;
+        margin: 0 0 0.75rem;
+        padding: 0;
+        margin-bottom: 1rem;
+    }
+
+    ul li {
+        margin-bottom: 0.375rem;
+        padding-left: 1.25rem;
+    }
+
+    ul li:before {
+        display: inline-block;
+        content: '';
+        border-radius: 0.375rem;
+        height: 0.75rem;
+        width: 0.75rem;
+        margin-right: 0.5rem;
+        background-color: ${neutral[86]};
+        margin-left: -1.25rem;
+    }
+
+    b {
+        font-weight: bold;
+    }
+`;
```

Comment:
This body styling looks familiar to the other atom, I imagine you're going to be duplicating it again, so it's worth working out how to share it. I have a feeling these atoms are going to be very much 'containers' for the abstracted body, footer, summary etc inside?